samples/web/content/apprtc/apprtc.py

```diff
@@ -312,14 +318,25 @@ def __str__(self):
 class Room:
   def __init__(self):
     self.clients = {}
+    self.allowed_clients = None;
```

Comment:
Good point. I wrote a test for the same reason but didn't think to add the documentation there.